test/unit/org/apache/cassandra/cql3/PreparedStatementsTest.java

```diff
@@ -581,4 +653,126 @@ private void testPrepareWithBatchLWT(ProtocolVersion version) throws Throwable
                       row(false, 1, 10, 20, null));
         assertEquals(rs.getColumnDefinitions().size(), 5);
     }
+
+    @Test
+    public void testPrepareWithAccordV4()
+    {
+        testPrepareWithAccord(ProtocolVersion.V4);
+    }
+
+    @Test
+    public void testPrepareWithAccordV5()
+    {
+        Assume.assumeTrue("Protocol v5 is CURRENT", ProtocolVersion.CURRENT != ProtocolVersion.V5);
+        testPrepareWithAccord(ProtocolVersion.V5);
+    }
+
+    @Test
+    public void testPrepareWithAccordCurrent()
+    {
+        testPrepareWithAccord(ProtocolVersion.CURRENT);
```

Comment:
nit: Maybe just parameterize on `SUPPORTED_VERSIONS`?

Reply:
100% agree but didn't want to rewrite all tests.

Reply:
JUnit 5 allows different tests to have different parameters, but not in JUnit 4 as its done via the constructor.